src/ApiGenerator/Generator/Razor/RazorGeneratorBase.cs

```diff
@@ -51,13 +49,13 @@ public abstract class RazorGeneratorBase
 			.EnableDebugMode()
             .Build();
 
-        protected async Task DoRazor<TModel>(TModel model, string viewLocation, string targetLocation, CancellationToken token)
+        protected static async Task DoRazor<TModel>(TModel model, string viewLocation, string targetLocation, CancellationToken token)
         {
             try
             {
                 token.ThrowIfCancellationRequested();
                 var generated = await Engine.CompileRenderAsync(viewLocation,  model);
-                WriteFormattedCsharpFile(targetLocation, generated);
+                await WriteFormattedCsharpFile(targetLocation, generated);
```

Comment:
This `await` is new, is it ok?

Reply:
These are in the code generator, it is because I changed the WriteFormattedCsharpFile method to be async, as I changed the formatter and from sync file write to async file write.

I can split these changes to the code generator out into a seperate PR if you like

Reply:
No need, I just want to make sure there's no negative end-user impact.